core/arch/arm/arm.mk

```diff
@@ -11,7 +11,9 @@ include mk/$(COMPILER_core).mk
 # Defines the cc-option macro using the compiler set for the core module
 include mk/cc-option.mk
 
+# Thread mutex support in core libtomcrypt
 CFG_LTC_OPTEE_THREAD ?= y
+
```

Comment:
Good opportunity to explain better, because honestly with only this I don't understand what `y` brings us and what limitation `n` would introduce...

Reply:
:)  okay

Reply:
I don't think this should describe what is protected by the mutexes used in LTC.

Reply:
> What do you think of:
> 
> ```makefile
> # Thread mutex support in core libtomcrypt. This must be enabled unless OP-TEE
> # runs a single thread using Libtomcrypt operations.
> CFG_LTC_OPTEE_THREAD ?= y
> ```
> 
> ?

OK, then how about this instead? 
```
ifneq ($(CFG_NUM_THREADS),1)
$(call force,CFG_LTC_OPTEE_THREAD,y)
endif
```

Reply:
If it's done like this we could just as well do this directly in `core/crypto.mk` to assign `_CFG_CORE_LTC_OPTEE_THREAD`.
I suppose we had this two-step configuration in case we wanted to change the default value.

> Maybe we should also consider the CFG_NUM_THREADS==0? (I.e. one building an optee for very specific reason not bringing thread support)

I'm not sure that makes any sense to have in mind upstream.

Reply:
> If it's done like this we could just as well do this directly in core/crypto.mk to assign _CFG_CORE_LTC_OPTEE_THREAD.
I suppose we had this two-step configuration in case we wanted to change the default value.

Agree.

> > Maybe we should also consider the CFG_NUM_THREADS==0? (I.e. one building an optee for very specific reason not bringing thread support)
> > 
> I'm not sure that makes any sense to have in mind upstream.

ok

Reply:
> > If it's done like this we could just as well do this directly in core/crypto.mk to assign _CFG_CORE_LTC_OPTEE_THREAD.
> > I suppose we had this two-step configuration in case we wanted to change the default value.
> 
> Agree.

+1